core/src/main/java/com/netflix/msl/io/MslEncoderFactory.java

```diff
@@ -114,7 +114,7 @@ static String quote(final String string) {
      */
     @SuppressWarnings("unchecked")
     static String stringify(final Object value) {
-        if (value == null || value.equals(null)) {
+        if (value == null) {
```

Comment:
This may exist to support JSONObject.Null and similar sentinel NULL objects. I should add a comment.